include/secp256k1_musig.h

```diff
@@ -317,12 +318,10 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_musig_pubkey_xonly_twea
  *  MuSig differs from regular Schnorr signing in that implementers _must_ take
  *  special care to not reuse a nonce. This can be ensured by following these rules:
  *
- *  1. Each call to this function must have a UNIQUE session_id32 that must NOT BE
+ *  1. Each call to this function must have a UNIQUE session_secrand32 that must NOT BE
  *     REUSED in subsequent calls to this function.
- *     If you do not provide a seckey, session_id32 _must_ be UNIFORMLY RANDOM
- *     AND KEPT SECRET (even from other signers). If you do provide a seckey,
- *     session_id32 can instead be a counter (that must never repeat!). However,
- *     it is recommended to always choose session_id32 uniformly at random.
+ *     If you do not provide a seckey, session_secrand32 _must_ be UNIFORMLY RANDOM
+ *     AND KEPT SECRET (even from other signers).
```

Comment:
We should then drop the condition "If you do not provide a seckey,"

Reply:
We could also add a reference to the new function, maybe below the numbered list.

something like "If you don't have access to good randomness, but you have access to a non-repeating counter, then see ..."

Reply:
Added both suggestions.